docs/tutorials/mobile/superres.md

```diff
@@ -0,0 +1,643 @@
+---
+title: Mobile super resolution
+description: Build Android and iOS applications for improving image resolution with built-in pre and post processing
+parent: Deploy on mobile
+grand_parent: Tutorials
+nav_order: 3
+---
+
+# Machine learning mobile application to improve image resolution
+{: .no_toc }
+
+Learn how to build an application to improve image resolution using ONNX Runtime Mobile, with a model that includes pre and post processing.
+
+You can use this tutorial to build the application for Android or iOS.
+
+The application takes an input input, performs the super resolution operation when the button is clicked and displays the image with improved resolution below, as in the following screenshot.
```

Comment:
nit: > The application takes an input input

Reply:
Fixed